client/snapshot.go

```diff
@@ -197,6 +197,30 @@ func (client *Client) SnapshotExport(setID uint64) (stream io.ReadCloser, conten
 		}
 		return nil, 0, fmt.Errorf("unexpected status code: %v", rsp.Status)
 	}
+	contentType := rsp.Header.Get("Content-Type")
+	if contentType != "application/x.snapd.snapshot" {
+		return nil, 0, fmt.Errorf("cannot import snapshot of content type %q", contentType)
```

Comment:
"import here" sound strange?  looking at the closest cases in client, this should be something like

"unexpected snapshot export content type %q"